pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -0,0 +1,151 @@
+// +build windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package winstats
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"sync"
+	"time"
+
+	cadvisorapi "github.com/google/cadvisor/info/v1"
+	"github.com/lxn/win"
+)
+
+// PerfCounterNodeStatsClient is a client that provides Windows Stats via PerfCounters
+type PerfCounterNodeStatsClient struct {
```

Comment:
Still prefer *not* exporting this type. See my comment in the original thread.

Reply:
Fixed (see original thread).